src/screens/Voucher/VoucherHomeScreen.tsx

```diff
@@ -12,11 +14,22 @@ import {
   Wrapper
 } from '../../components';
 import { colors, texts } from '../../config';
+import { addToStore, readFromStore } from '../../helpers';
+import {
+  storeVoucherAuthToken,
+  storeVoucherMemberId,
+  storeVoucherMemberLoginInfo,
+  voucherMemberLoginInfo
+} from '../../helpers/voucherHelper';
 import { useStaticContent, useVoucher } from '../../hooks';
-import { ScreenName } from '../../types';
+import { logIn } from '../../queries/vouchers';
+import { ScreenName, VoucherLogin } from '../../types';
+
+const SAVED_DATE = 'savedDate';
```

Comment:
this is very generic. i suggest including some naming with member login to have it thematised

Reply:
This is long, but I think it's clearer.
```
const SAVED_DATE_OF_LAST_ACCOUNT_CHECK = 'savedDateOfLastAccountCheck';
```